src/dcttestlibs/dcsstubs.lua

```diff
@@ -69,15 +69,13 @@ _G.env = env
 
 local timer = {}
 function timer.getTime()
-	-- 00:03:54 model time
-	return 234
+	return os.clock()
```

Comment:
Instead of tying to `os.clock` would providing another interface to update the model time as the test deems appropriate be better?

Something like:
```
local model_time = 0
function timer.stub_setTime(val)
    model_time = val
end

function timer.getTime()
    return model_time
end
```

Reply:
I haven't checked if the tests run on separate interpreters, but if they do, it's possible. It would allow times to make sense in the test cases too, instead of picking the magic values that make them happy (especially in regards to Theater.exec time vs timer.getTime time).

Reply:
Resolved in https://github.com/jtoppins/dct/pull/180/commits/bd6168f8c29bdde5bb0f38f3abea4a19036fd29c